---
#
# By default, content added below the "---" mark will appear in the home page
# between the top bar and the list of recent posts.
# To change the home page layout, edit the _layouts/home.html file.
# See: https://jekyllrb.com/docs/themes/#overriding-theme-defaults
# <h3 class="blackpar_title">(Models, Training and Inference)</h3>
layout: home
permalink: /
---
<div style="display: flex; align-items: center; justify-content: center; background: url('images/header.jpg') no-repeat; background-size: cover; user-select: none; height: 600px; padding: 0;">
    <h1 class="blackpar_title" style="text-align: center; font-weight: bold; line-height: 1.2; text-shadow: 0px 0px 5px black; color:#CCCCFF">Machine Learning for Genomics Explorations (MLGenX)</h1>
</div>


<!--starts inverted2 colors-->
<div class="inverted2">

<br>
<p>
Our limited understanding of the biological mechanisms underlying diseases remains a critical
bottleneck in drug discovery. As a result, we often lack insights into why patients develop specific conditions,
leading to the failure of many drug candidates in clinical trials. Recent advancements in genomics platforms
and the emergence of diverse omics datasets have sparked increasing interest in this field. The primary
objective of this workshop is to bridge the gap between machine learning and genomics, emphasizing target
identification and emerging drug modalities such as gene and cell therapies and RNA-based drugs. By
fostering interdisciplinary collaboration, we aim to advance the integration of these disciplines and accelerate
innovation in drug discovery.</p>
<br>
</div>

<!--starts inverted colors-->
<div class="inverted">
<br>

<br>
<!-- Call for Papers -->
<h2 class="blackpar_title" id="call_for_papers">Call for Papers</h2>

<p>
This year, the workshop will feature three distinct tracks designed to welcome a diverse array of researchers
in the field of machine learning and biology: the Main Track including application and ML topics, the
Special Track on LLMs and Agentic AI, and the Tiny Papers Track. Papers in the main and the special tracks must be prepared and submitted as a single file: 8 pages for the paper, with unlimited pages for references, the impact statement, and appendices.
</p>
<p>
Both contributions introducing new ML methods to existing problems and those that highlighting and explaining open problems are welcome. 
We also encourage submissions related to application of molecular biology, including but not limited to, single-cell RNA analysis, bulk RNA studies, proteomics, and microscopy imaging of cells and/or tissues.
</p>
<p>
We consider a broad range of subject areas including but not limited to the following topics.
</p>

<!--
<ul>
    <li>
	<u><b>Biological sequence design</b></u>: Prediction and optimization of biological sequences, incorporating constraints and prior knowledge
    </li>
	<ul>
	    <li>
	        Effectively miniaturize DNA/RNA/Protein sequences while preserving their key properties
	    </li>
	    <li>
	        Multi-omics-based sequence design
	    </li>
	    <li>
		Modeling long-range sequence interactions
	    </li>
	    <li>
		Tissue/cell-type specific sequence design
	    </li>
	</ul>

    <li>
	<u><b>Inferring cellular communication via cell states and organization in tissues</b></u>: Causal representation learning to model cell states and cellular communities 
    </li>
	<ul>

	    <li>
		Multi-omics data integration (single cell, spatial transcriptomics)
	    </li>
	    <li>
		Cell-cell interactions inference
	    </li>
	    <li>
		Mechanistic modeling of cells in their context to infer cellular function
	    </li>
	    <li>
		Modeling long-range interactions in single-cell and spatial omics
	    </li>
	</ul>

    <li>
	<u><b>Perturbative biology</b></u>: Interpretable and foundation models to understand cellular responses to perturbations
    </li>
	<ul>

	    <li>
		Translating genetic perturbations to understandable and actionable molecular changes
	    </li>
	    <li>
		Causal reasoning for learning gene regulatory networks
	    </li>
	    <li>
		Integrating multimodal perturbation readouts (transcriptomic and phenotypic) to better characterize the broader molecular effects
	    </li>
	    <li>
		Large-scale foundation models for predicting transcriptional outcomes of novel perturbations
	    </li>
	    <li>
		Generalizability of perturbation predictive models across cell lines and cellular contexts
	    </li>
	</ul>
</ul>

<h5>From first principles: AI for Genomics Exploration</h5>
-->
<p><b>Main Track</b>:</p>
<ul>
    <li>Foundation models for genomics
    </li>
    <li>Biological sequence design
    </li>
    <li>Interpretability and Generalizability in genomics
    </li>
    <li>Causal representation learning
    </li>
    <li>Perturbation biology
    </li>
    <li>Modeling long-range dependencies in sequences, single-cell and spatial omics
    </li>
    <li>Integrating multimodal perturbation readouts
    </li>
    <li>Active learning in genomics
    </li>
    <li>Generative models in Biology
    </li>
    <li>Multimodal representation learning
    </li>
    <li>Uncertainty quantification
    </li>
    <li>Optimal transport
    </li>
    <li>Experimental design for Biology
    </li>
    <li>Graph neural network and knowledge graph
    </li>
    <li>New datasets and benchmarks for genomics explorations
    </li>
</ul>

<p><b>Special Track on LLMs and Agentic AI</b>:</p>
<ul>
    <li>Pre-training multi-omics models
    </li>
    <li>Synthetic data generation and data quality for pre-training, fine-tuning and instruction tuning
    </li>
    <li>Fine-tuning (SFT, RLHF, RL with lab feedback, ...) on novel tasks
    </li>
    <li>In-context learning with large-context models
    </li>
    <li>Reasoning through prompt engineering or architectural design
    </li>
    <li>Interpretability and uncertainty quantification
    </li>
    <li>Knowledge retrieval (RAG, knowledge graph, ...)
    </li>
    <li>Efficient interactive system designs (agents, humans, and biological tools)
    </li>
    <li>Training/fine-tuning LLM-powered design and planning engine
    </li>
</ul>

<p><b>Tiny Papers Track</b>:</p>
<p>This year, ICLR is discontinuing the separate “Tiny Papers” track, and is instead requiring each workshop to accept short (3-4 pages in ICLR format) paper submissions, with an eye towards inclusion; see ​​<a href="https://iclr.cc/Conferences/2025/CallForTinyPapers">https://iclr.cc/Conferences/2025/CallForTinyPapers</a> for more details. Authors of these papers will be earmarked for potential funding from ICLR, but need to submit a separate application for Financial Assistance that evaluates their eligibility. This application for Financial Assistance to attend ICLR 2025 will become available on <a href="https://iclr.cc/Conferences/2025/">https://iclr.cc/Conferences/2025/</a> at the beginning of February and close on March 2nd.</p>



<h3 class="blackpar_title">Submission Instructions</h3>
<p>
Similar to the main ICLR conference, submissions will be double blind.  
We use <a href="https://openreview.net/group?id=ICLR.cc/2025/Workshop/MLGenX#tab-your-consoles">OpenReview</a> to host papers. There will be a strict upper limit of 8 pages for the main text of the submission in the main and special tracks, and 4 pages for the main text of the submission in the tiny paper track, with unlimited additional pages for citations and appendices. To prepare your submission, please use the <a href="https://github.com/ICLR/Master-Template/raw/master/iclr2025.zip">ICLR template style</a>.
</p>
<p>
Submissions that are identical to versions that have been previously published, or accepted to the main ICLR conference are not allowed. However, papers that cite previous related work by the authors and papers that have appeared on non-peer reviewed websites (like arXiv) do not violate the policy. Submission of the paper to archival repositories such as arXiv is allowed during the review period.
</p>
<p>
<b>Note:</b><span style="color: #D2042D"> Authors are permitted to submit works that are currently under review by other venues. Additionally, accepted papers are not considered archival and can be subsequently published in other conferences or journals.</span> 
</p>
<p>
We plan to offer <b>Best Paper</b> Award(s), and exceptional submissions will be chosen for oral presentations. Please note that while our workshop is not archival, accepted papers will be featured on the workshop website.
</p>

<p> <b>Note:</b> Official reviews are anonymous, and unlike the main  conference, the papers and reviews are not made public until acceptance!</p>


<h3 class="blackpar_title">Camera Ready Instructions</h3>
<p>
An email with instructions for uploading camera-ready submissions will go out mid-March 2025. To prepare your camera ready, please use the <a href="https://github.com/Namkyeong/mlgenx.github.io/raw/refs/heads/main/latex_templates/MLGenX_Formatting_Instructions.zip" download>MLGenX 2025 template style</a>.
</p>
<ul>
    <li>The authors can use one additional page beyond the page limit specified during the submission (9 pages for main/special track papers and 5 pages for tiny track papers). This extra page can be used to appropriately address the comments received during the review process.
    </li>
</ul>
<!--
<h3 class="blackpar_title">Call for Reviewers</h3>
<p>
We are looking for reviewers with expertise in machine learning or computational biology. If you are interested in joining us, kindly complete this <a href="https://forms.gle/tZoUCQJXYHVjSe6Z8">form</a>.
</p> -->

<h3 class="blackpar_title" id="dates">Important Dates</h3>
<p>
All deadlines are 11:59 pm UTC -12h ("Anywhere on Earth"). All authors must have an OpenReview profile when submitting.
<p>
<ul>
    <li>Submission Deadline (<b>Main and Special Tracks</b> - Up to 8 pages): <del> February 12, 2025 </del> <span style="color: #D2042D"> February 16, 2025 </span></li>
    <li>Submission Deadline (<b>Tiny Papers Track</b> - Up to 4 pages): <span style="color: #D2042D">February 23, 2025 </span></li>
    <li>Acceptance Notification: March 5, 2025 </li>
    <li>Camera-Ready Deadline: April 24, 2025 </li>
    <li>Workshop Date: Sunday, April 27, 2025 </li>
</ul>
</p>

<!--
<h3 class="blackpar_title">Workshop Registration</h3>
<p>
Whether you're a seasoned professional or a curious enthusiast, all are welcome to attend! Don't worry if you don't have an accepted paper – participation is open to everyone.</p>

<p> If you have already registered for ICLR, you can join us at the MLGenX workshop. However, if you're solely interested in the workshop, you can still participate in the MLGenX workshop by registering for the "Saturday Workshop 1 Day Pass". Please visit <a href="https://iclr.cc/Register/view-registration">this link</a> to secure your spot. </p>

<p>We look forward to meeting you in Singapore!
</p> -->

<!-- Speakers -->
<br>
<h2 class="blackpar_title" id="speakers">Tentative Speakers & Panelists</h2>
<p>
{% include speakers.html %}
</p>

<!-- Organizers -->
<h2 class="blackpar_title" id="organizers">Organizers</h2>
<p>
{% include organizers.html %}
</p>

<!-- Organizers -->
<h2 class="blackpar_title" id="students">Student Organizers</h2>
<p>
{% include students.html %}
</p>


<!--
<h2 class="blackpar_title">Sponsor</h2>
<div class="row">
    <div class="col">
        <center>
            <img src="">
        </center>
    </div>
    <div class="col">
        <center>
            <img src="" width="250px">
        </center>
    </div>
</div>-->


<!--ends inverted colors-->
<!-- Default Statcounter code for mlgenx
https://mlgenx-workshop.github.io/ -->
<script type="text/javascript">
var sc_project=12885210;
var sc_invisible=1;
var sc_security="21af2424";
</script>
<script type="text/javascript"
src="https://www.statcounter.com/counter/counter.js"
async></script>
<noscript><div class="statcounter"><a title="Web Analytics"
href="https://statcounter.com/" target="_blank"><img
class="statcounter"
src="https://c.statcounter.com/12885210/0/21af2424/1/"
alt="Web Analytics"
referrerPolicy="no-referrer-when-downgrade"></a></div></noscript>
<!-- End of Statcounter Code -->
